keywords/MobileRestClient.py

```diff
@@ -2266,3 +2266,53 @@ def get_revs_num_in_history(self, url, db, doc_id, auth=None):
         doc = self.get_doc(url, db, doc_id, auth)
         logging.debug(doc)
         return doc["_revisions"]["ids"]
+
+    def sgCollect_restCall(self, sg_host, redact_level=None, redact_salt=None, output_directory=None, upload=False, upload_host=None, customer=None, ticket=None):
+        """
+        set all parameters to the dictionary and do post
+        """
+        body = {}
+        body["upload"] = upload
+        if redact_level is not None:
```

Comment:
change this to body["redact_level"] = redact_level, no need for if condition. If it's none then it will become none, if anything else then it will become that. Same as for below params for dictionary

Reply:
Here I am adding params to the body only if variables are none. I do not pass these values to rest call if they are none. If I set to None, key will be set and value will be None. So, I thought having If condition to check for None is better

Reply:
explanation seems OK.